cogs5e/sheets/gsheet.py

```diff
@@ -369,12 +369,12 @@ def get_skills_and_saves(self):
             else:
                 profcell = None
             try:
-                value = int(character.value(cell))
+                value = int(character.value(cell)) + int("C" in cell and (self.version == 2.1 and character.value("AR58") or self.version == 2 and character.value("AQ26"))) + int("C" in cell and is_joat and int(character.value("H14"))//2)
```

Comment:
Why do the check again, instead of just reading `is_joat`?

Reply:
It does check `is_joat` where it needs to in there. The first plus pulls in the globally applied bonus.

Reply:
Can you give an explanation of exactly what this is doing? Why do we need to check all these additional cells instead of just reading the final bonus presented on the sheet?

Reply:
This is taking the values of the ability modifiers (`"C"in cell`) which aren't a "final bonus" and adding the bonus that applies to all ability checks from things like a Stone of Good Luck (the value in AR58 for 2.1 or AQ26 for 2.0) and then adding the JoAT half proficiency bonus if applicable. This will make it consistent with how Beyond handles things and make it so you won't need a `joat` or `ls` snippet to apply Jack of All Trades or Luck Stone bonus to non-skill ability checks. Currently, everything but the straight ability checks include all this as presented from the sheet, but the only location for it to pull for base ability checks is the ability modifier cell which doesn't/shouldn't include these bonuses.

Reply:
For readability, can you pull base ability checks into its own block (might require changing some of the constants too) rather than doing it in a one-liner? Making some kind of loop dedicated to something like `BASE_ABILITY_CHECKS` makes sense to handle those special cases.